docs/UserGuide.md

```diff
@@ -11,117 +11,287 @@
-2. Copy the file to the folder you want to use as the home folder for your Address Book.
+   Click [here](http://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html) to download the latest Java version.
+   
+1. Download the latest `tars.jar` from the 'releases' tab. [*No releases available yet*]
+2. Copy the file to the folder you want to use as the home folder for your TARS App.
 3. Double-click the file to start the app. The GUI should appear in a few seconds. 
-   > <img src="images/Ui.png" width="600">
-
 4. Type the command in the command box and press <kbd>Enter</kbd> to execute it. <br>
    e.g. typing **`help`** and pressing <kbd>Enter</kbd> will open the help window. 
 5. Some example commands you can try:
-   * **`list`** : lists all contacts
-   * **`add`**` John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01` : 
-     adds a contact named `John Doe` to the Address Book.
-   * **`delete`**` 3` : deletes the 3rd contact shown in the current list
+   * **`ls`** : lists all tasks
+   * **`add`**` Complete CS2103 Quiz 3 -dt 23/09/2016 -p h -t Quiz -t CS2103` : 
+     adds a task `Complete CS2103 Quiz 3` to TARS.
+   * **`del`**` 3` : deletes the 3rd task shown in TARS.
    * **`exit`** : exits the app
 6. Refer to the [Features](#features) section below for details of each command.<br>
 
 
 ## Features
 
-#### Viewing help : `help`
+#### Displaying a list of available commands : `help`
+Displays a list of available commands with their individual usage information <br>
 Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
  
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
+#### Adding a task : `add`
+Adds a task to TARS<br>
+Format: `add <TASK> -dt <START_DATE/TIME> to <END_DATE/TIME> -p <PRIORITY> -t <TAG>[, <TAG>, <TAG>,...] -r <NUM_TIMES> <FREQUENCY>` 
  
-> Words in `UPPER_CASE` are the parameters, items in `SQUARE_BRACKETS` are optional, 
-> items with `...` after them can have multiple instances. Order of parameters are fixed. 
+> Words in `UPPER_CASE` are the parameters. Other than `<TASK>`, all parameters are optional. 
+> Order of parameters are not fixed.
+>
+> Time is in a 24 hour format (e.g. `1330`) 
 > 
-> Persons can have any number of tags (including 0)
+> Priority options are: `h` for High, `m` for Medium, `l` for Low
 
 Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe p/1234567 e/betsycrowe@gmail.com a/Newgate Prison t/criminal t/friend`
+* `add Meet John Doe -dt 26/09/2016 0900 to 26/09/2016 1030 -t CATCH UP`
+* `add Complete CS2103 Quiz -dt 23/09/2016 -p h, -t Quiz -t CS2103, -r 13 EVERY WEEK`
 
-#### Listing all persons : `list`
-Shows a list of all persons in the address book.<br>
-Format: `list`
+#### Reserving timeslots for a task : `rsv`
```

Comment:
👍 I like that your have this feature planned in already ~ however as mentioned remember to complete ALL must have before looking at nice to have ~ (and yupp i know most of the must have might/might not have been completed, I'm just point it out so that you don't have to be over stressed to implement everything ~)